Backend/bicep/main.bicep

```diff
@@ -0,0 +1,54 @@
+targetScope='subscription'
+
+// The main bicep module to provision Azure resources.
+// For a more complete walkthrough to understand how this file works with azd,
+// see https://learn.microsoft.com/azure/developer/azure-developer-cli/make-azd-compatible?pivots=azd-create
+
+var abbrs = loadJsonContent('./abbreviations.json')
+
+// Optional parameters to override the default azd resource naming conventions.
+// Add the following to main.parameters.json to provide values:
```

Comment:
My intent is to leave _some_ breadcrumbs, but I don't want to turn this into a bicep tutorial. Should I pare these comments down more?